### PR TITLE
Website Redesign Part I

### PR DESCRIPTION
**Pull request by lukasbischof**

### CODE REVIEW

**Comment by lukasbischof** [comment]
Partially implements #209

File: app/javascript/images/index.js [hunk 1]
```diff
@@ -1,2 +1,2 @@
-import './theme';
-import './my_zivi_transparent.png';
+require.context('.', true);
+require.context('./home', true);
```
Comment (TheFehr) [inline app/javascript/images/index.js:2]: Why?

Comment (lukasbischof) [reply app/javascript/images/index.js:2]: why not

**Review by TheFehr — APPROVED** [review]
The mobile screenshot seems a bit off. Otherwise looks good

**Pull request merged**